### PR TITLE
Update .env logic for F2F

### PR DESCRIPTION
**Pull request by Z-Mohamoud**
### What changed

Update logic to use local .env files

### Issue tracking

- [KIWI-1667](https://govukverify.atlassian.net/browse/KIWI-1667)

### Environment variables or secrets

- [ ] Documented in the README
- [ ] Added screenshots to show the implementation is working
- [ ] Ran cfn-lint on any SAM templates

### Other considerations

[KIWI-1667]: https://govukverify.atlassian.net/browse/KIWI-1667?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -6,4 +6,11 @@ UNIVERSAL_ANALYTICS_GTM_CONTAINER_ID="GTM-TK92W68"
 GOOGLE_ANALYTICS_4_GTM_CONTAINER_ID="GTM-KD86CMZ"
 GA4_ENABLED="true"
 ANALYTICS_DOMAIN="localhost"
-LOG_LEVEL="debug"
\ No newline at end of file
+LOG_LEVEL="debug"
+
+# QA Env
+IPV_STUB_URL=
+IPV_BASE_URL=
```
Comment (Caitlin-cooling) [inline .env.sample:13]: What is the difference between these 2 vars?

Comment (Z-Mohamoud) [reply .env.sample:13]: IPV_STUB_URL is the URL used to start the CIC stub service (https://f2f-ipv-stub-ipvstub.review-o.dev.account.gov.uk/start)

IPV_BASE_URL is the baseURL for the F2F FE (https://f2f-cri-front.review-o.dev.account.gov.uk)

File: .env.sample [hunk 2]
```diff
@@ -6,4 +6,11 @@ UNIVERSAL_ANALYTICS_GTM_CONTAINER_ID="GTM-TK92W68"
...
-LOG_LEVEL="debug"
\ No newline at end of file
+LOG_LEVEL="debug"
+
+# QA Env
+IPV_STUB_URL=
+F2F_FE_BASE_URL=
+TEST_HARNESS_URL=
+SESSION_TABLE=
+F2F_API_URL=
```
Comment (Caitlin-cooling) [inline .env.sample:16]: Why are we not able to use the `API_BASE_URL` that already exists?

File: .gitignore [hunk 3]
```diff
@@ -15,7 +15,8 @@ yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
 
-# Additional ignores
+# Additional ignores.
+.env
```
Comment (Caitlin-cooling) [inline .gitignore:19]: Nice 👍

File: .env.sample (hunk 2 above)
Comment (Z-Mohamoud) [reply .env.sample:16]: API_BASE_URL is used for localhost testing I believe, but we can remove from .env.sample as it's set in the package.json and run-tests.sh for test runs it's required for

Comment (Z-Mohamoud) [reply .env.sample:16]: The API_BASE_URL is set to a localhost URL which causes trouble with local test runs. So API_BASE_URL was introduced so we can reference the F2F BE URL for calls to the /callback endpoint

Comment (Caitlin-cooling) [reply .env.sample:16]: What is the localhost url used for? Thought everything was pointing at dev envs now?

File: .env.sample [hunk 4]
```diff
@@ -6,4 +6,11 @@ UNIVERSAL_ANALYTICS_GTM_CONTAINER_ID="GTM-TK92W68"
...
-LOG_LEVEL="debug"
\ No newline at end of file
+LOG_LEVEL="debug"
+
+# QA Env
+IPV_STUB_URL=
+F2F_FE_BASE_URL=
+TEST_HARNESS_URL=
+SESSION_TABLE=
+API_BASE_URL=
```
Comment (Caitlin-cooling) [inline .env.sample:16]: Do we need `API_BASE_URL` twice or can we remove one?

File: .env.sample (hunk 2 above)
Comment (Z-Mohamoud) [reply .env.sample]: Updated logic to remove all localhost references and use API_BASE_URL in tests

**Pull request merged**